exercises/gigasecond/GigasecondTests.elm

```diff
@@ -0,0 +1,49 @@
+port module Main exposing (..)
+
+import Test.Runner.Node exposing (run)
+import Json.Encode exposing (Value)
+import Test exposing (..)
+import Expect
+import Date
+import Gigasecond exposing (add)
+
+
+tests : Test
+tests =
+    describe "Gigasecond"
+        [ describe "add"
+            [ test "2011-04-25" <|
+                \() ->
+                    Expect.equal (actual "2011-04-25") (expected "2043-01-01T01:46:40")
```

Comment:
I really appreciate that you've tried to tighten this test format up, but I wonder if it might be a bit too much indirection? Especially in a learning context, I think it's best that the user see the direct application of their function. In fact, in this case the wording almost looks like you're comparing `"2011-04-25"` to `"2043-01-01T01:46:40"` as literals...

In this testing situation, I'd accept a "risky" date function to add clarity. Something like (untested)...

```
date : String -> Date.Date
date input =
    case Date.fromString input of
        Ok date ->
            date
        Err reason ->
            Debug.crash reason
```

Then the tests could be something like

`Expect.equal (Gigasecond.add (date "1977-06-13")) (date "2009-02-19T01:46:40")`